#the header of the message
Header header

float64 brake
float64 dir
float64 speed
float64 x
float64 y
float64 orientation
float64 distance_traveled
int32 turnright
int32 turnleft
int32 headlights
int32 taillights
int32 reverselights
int32 cross_sensor
int32 vert_sign
int32 errors